frontend/widgets/VolumeMeter.cpp

```diff
@@ -714,18 +714,17 @@ void VolumeMeter::paintHMeter(QPainter &painter, int x, int y, int width, int he
 	int errorLength = maximumPosition - errorPosition;
 	locker.unlock();
 
-	if (clipping) {
-		peakPosition = maximumPosition;
-	}
+	if (clipping)
+		peak = clipLevel + 0.1f;
```

Comment:
```suggestion
	if (clipping) {
		peak = clipLevel + 0.1f;
	}
```
I prefer (and think it's good practice) to demarcate blocks explicitly with curly braces even for single instructions (the changed code did so already).